### DISCUSSION TITLE
SoH increased !!!

### DISCUSSION THREAD

**Discussion by vineethktpla** [flobz/psa_car_controller#852] [Q&A]
Im tracking my car's SoH through home assistant using psa car controller provided api. It was showing 95% till today, but all of a sudden now it's reading 100%.
So my question is how reliable this data is. I hope it's coming from car's ECU
![Screenshot_20240415-201846](https://github.com/flobz/psa_car_controller/assets/146456863/ae8a367c-9e80-4ff1-95f3-9987d3d64910)

**Comment by GeraldPape** [comment]
Tried the API call for my Citroen C5, returns 0. Seems not all Cars provide that info.
@flobz would it make sense to start a page to document what works with what car model?
